### PR TITLE
Servicenow ocean docs

### PR DESCRIPTION
**Pull request by PeyGis**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

## Added docs pages

Please also include the path for the added docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

## Updated docs pages

Please also include the path for the updated docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md [hunk 1]
```diff
@@ -0,0 +1,825 @@
...
+<TabItem value="real-time-always-on" label="Real Time & Always On" default>
+
+Using this installation option means that the integration will be able to update Port in real time using webhooks.
+
+This table summarizes the available parameters for the installation.
+Set them as you wish in the script below, then copy it and run it in your terminal:
+
+| Parameter                                | Description                                                                                                   | Required |
+| ---------------------------------------- | ------------------------------------------------------------------------------------------------------------- | -------- |
+| `port.clientId`                          | Your port client id                                                                                           | ✅       |
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md:40]:
```suggestion
| `port.clientId`                          | Your Port client id                                                                                           | ✅       |
```

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md [hunk 2]
```diff
@@ -0,0 +1,825 @@
...
+| `port.clientSecret`                      | Your port client secret                                                                                       | ✅       |
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md:41]:
```suggestion
| `port.clientSecret`                      | Your Port client secret                                                                                       | ✅       |
```

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md (hunk 1 above)
Comment (MPTG94) [reply docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md:40]: We usually include in this table that shows you where to get the creds, please include it

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md [hunk 3]
```diff
@@ -0,0 +1,825 @@
...
+| `port.baseUrl`                           | Your port base url, relevant only if not using the default port app                                           | ❌       |
+| `integration.identifier`                 | Change the identifier to describe your integration                                                            | ✅       |
+| `integration.type`                       | The integration type                                                                                          | ✅       |
+| `integration.eventListener.type`         | The event listener type                                                                                       | ✅       |
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md:45]: This is usually part of the advanced configuration section

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md [hunk 4]
```diff
@@ -0,0 +1,825 @@
...
+| `OCEAN__INTEGRATION__CONFIG__SERVICENOW_USERNAME` | The ServiceNow account username                                                                                    | ✅       |
+| `OCEAN__INTEGRATION__CONFIG__SERVICENOW_PASSWORD` | The ServiceNow account password                                                                                    | ✅       |
+| `OCEAN__INTEGRATION__CONFIG__SERVICENOW_URL`      | The ServiceNow instance URL                                                                                        | ✅       |
+| `OCEAN__INITIALIZE_PORT_RESOURCES`                | Default true, When set to false the integration will not create default blueprints and the port App config Mapping | ❌       |
+| `OCEAN__INTEGRATION__IDENTIFIER`                  | Change the identifier to describe your integration, if not set will use the default one                            | ❌       |
+| `OCEAN__PORT__CLIENT_ID`                          | Your port client id                                                                                                | ✅       |
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md:89]:
```suggestion
| `OCEAN__PORT__CLIENT_ID`                          | Your Port client id                                                                                                | ✅       |
```

Comment (MPTG94) [reply docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md:89]: Same comment as before about linking the creds guide

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md [hunk 5]
```diff
@@ -0,0 +1,825 @@
...
+| `OCEAN__PORT__CLIENT_SECRET`                      | Your port client secret                                                                                            | ✅       |
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md:90]:
```suggestion
| `OCEAN__PORT__CLIENT_SECRET`                      | Your Port client secret                                                                                            | ✅       |
```

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md [hunk 6]
```diff
@@ -0,0 +1,825 @@
...
+  Note the value of the `blueprint` key - if you want to use a hardcoded string, you need to encapsulate it in 2 sets of quotes, for example use a pair of single-quotes (`'`) and then another pair of double-quotes (`"`)
+  :::
+
+### Ingest data into Port
+
+To ingest ServiceNow objects using the [integration configuration](#configuration-structure), you can follow the steps below:
+
+1. Go to the DevPortal Builder page.
+2. Select a blueprint you want to ingest using ServiceNow.
+3. Choose the **Ingest Data** option from the menu.
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md:296]: This is outdated, the guide should send the user to the data sources page and instruct him to add a new data source

**Review by MPTG94 — APPROVED** [review]
I left a few comments, please take care of them before merging

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md (hunk 6 above)
Comment (PeyGis) [reply docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md:296]: Thanks Mor. Can you send me a reference to the new guide on this section?

Comment (MPTG94) [reply docs/build-your-software-catalog/sync-data-to-catalog/incident-management/servicenow.md:294]: 1. Go to the DevPortal Builder page.
2. Select the Data Sources tab at the left sidebar.
3. Click on `+ Data Source` at the top right corner.
4. Select ServiceNow under the Incident Management category.
5. Modify the [configuration](#configuration-structure) according to your needs.
6. Run the installation command.
7. Click `Next` and you can view the integration configuration and update it as necessary.

**Review by hadar-co — APPROVED** [review]

**Pull request merged**